src/app/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.scss

```diff
@@ -0,0 +1,8 @@
+.box {
+  box-shadow: 0 6px 6px rgba(0, 0, 0, 0.25);
+}
+
+html,
+body {
+  background-color: var(--ubs-sextiary-light-grey);
+}
```

Comment:
not sure we should have this here
if this is background color it should be set globally for UBS admin

Reply:
Fixed